core/src/main/java/extensions/ListExt.kt

```diff
@@ -0,0 +1,9 @@
+package com.example.core.extensions
+
+fun <T> MutableList<T>.addFlatLists(vararg collections: Collection<List<T>>): MutableList<T> {
+    return this.apply {
+        collections.forEach {
+            addAll(it.flatten())
+        }
+    }
```

Comment:
you dont need this func
you can use flatten operator

Reply:
ok